packages/wrangler/src/init.ts

```diff
@@ -252,7 +253,7 @@ export async function initHandler(args: InitArgs) {
 				c3Arguments.unshift("-y"); // arg for npx
 			}
 
-			c3Arguments.unshift(...getC3CommandFromEnv().split(" "));
+			c3Arguments.unshift(...shellquote.parse(getC3CommandFromEnv()));
```

Comment:
This would have failed in scenarios where the c3 command was pointing to a local executable, for example, in a directory with spaces